lib/sqlalchemy/testing/requirements.py

```diff
@@ -1096,4 +1096,27 @@ def _has_cextensions(self):
 
     @property
     def computed_columns(self):
+        "Supports computed columns"
+        return exclusions.closed()
+
+    @property
+    def computed_columns_stored(self):
+        "Supports computed columns with `persisted=True`"
+        return exclusions.closed()
+
+    @property
+    def computed_columns_virtual(self):
+        "Supports computed columns with `persisted=False`"
+        return exclusions.closed()
```

Comment:
this must be two separate since some backend, like firebird, may not support neither of them